src/ui/states/state-needs-auth-info.svelte

```diff
@@ -32,86 +33,100 @@
   });
 </script>
 
-<form on:submit|preventDefault={handleContinue}>
-  <ModalHeader>Billing email address</ModalHeader>
-  <ModalSection>
-    <div class="form-container">
-      <div class="form-label"><label for="email">Email</label></div>
-      <div class="form-input {inputClass}">
-        <input
-          name="email"
-          placeholder="john@appleseed.com"
-          autocapitalize="off"
-          bind:value={email}
-        />
-      </div>
-      {#if error}<div class="form-error">{error}</div>{/if}
-    </div>
-  </ModalSection>
-  <ModalFooter>
-    <RowLayout>
-      <Button disabled={processing}>
-        {#if processing}
-          <ProcessingAnimation />
-        {:else}
-          Continue
+<div class="container">
+  <ModalHeader>
+    <span>Billing email address</span>
+    <CloseButton on:click={onClose} />
+  </ModalHeader>
+  <form on:submit|preventDefault={handleContinue}>
+    <ModalSection>
+      <div class="form-container">
+        <div class="form-label"><label for="email">Email</label></div>
+        <div class="form-input {inputClass}">
+          <input
+            name="email"
+            placeholder="john@appleseed.com"
```

Comment:
NABD and it was already like this but should we change this to something else like `Email`?